### ISSUE TITLE
admin Not Found

### ISSUE THREAD

**Issue by Moorcam** [HansSchouten/PHPageBuilder#179]
After install, I go to localhost/pagebuilder/admin and get a 404 Not found

**Comment by mrdecoder** [comment]
hi because this works with composer stuff the normal way for opening stuff not apply 
this is also why i hate composer things 

but try to just do /admin 
instead of /pagebuilder/admin
make sure you have a .htacces and index.php

**Comment by Moorcam** [comment]
> hi because this works with composer stuff the normal way for opening stuff not apply this is also why i hate composer things
> 
> but try to just do /admin instead of /pagebuilder/admin make sure you have a .htacces and index.php

Using pagebuilder/admin because it's hosted under pagebuilder.
I too hate Composer
Have a index.php but not .htaccess

**Comment by mrdecoder** [comment]
yes thats ok but then you have a phpagebuilder folder inside pagebuilder 
else it not work 
again this is cause of composer BS

i had same problem so atm for testing and rebuilding the whole thing i just added it to a stand alone website directly on root zo its not in a folder 

in case you do want it in a folder right now then maybe you can try a subdomain 
cause that does work normaly with composer stuff 

but atm what you basicly need is a folder called pagebuilder 
then in this folder you need the index.php and the .htaccess 
and you need to have the phpagebuilder folder 
the index,php is basicly the file that runs all

**Comment by Moorcam** [comment]
Where is the index.php that I need? It didn't come with the download

**Comment by mrdecoder** [comment]
try downloading the boilerplate 
https://github.com/HansSchouten/PHPagebuilder-boilerplate
this again is cause of the composer 

add that and ull be good to go i think
